### DISCUSSION TITLE
Geo Spatial Data Field

### DISCUSSION THREAD

**Discussion by i-jared** [MichalLytek/type-graphql#1318] [Q&A]
I'm using PostGIS and Geojson to incorporate location into my data (using typeorm and postgres). I want to return Geojson types through graphql, but I can't find any compatible type in typegraphql. 

Am I missing something? How can can I define the type for graphql to return?

**Comment by MichalLytek** [comment]
You can use any GraphQL scalar with TypeGraphQL, even create your own.
